rm.Extensions/DateTimeExtension.cs

```diff
@@ -50,5 +51,36 @@ public static DateTime AsUtcKind(this DateTime datetime)
 		{
 			return DateTime.SpecifyKind(datetime, DateTimeKind.Utc);
 		}
+
+		/// <summary>
+		/// Returns the date in year month ordinal format
+		/// </summary>
+		public static string ToYearMonthOrdinal(this DateTime date)
```

Comment:
It's incorrect to write a date as `February 11th, 2011`; `February 11, 2011` is correct.

Reply:
If I were to write this method, I would write an `int.Ordinal()` extension method and use it here.
```
ordinalMap = new map() {
	{ 11, "th" },
	{ 12, "th" },
	{ 13, "th" },
	{  1, "st" },
	{  2, "nd" },
	{  3, "rd" },
}

toOrdinal(n):
	n = abs(n)
	digit21 = n %100
	if ordinalMap.has(digit21):
		return ordinalMap[digit21]
	digit1 = n %10
	if ordinalMap.has(digit1):
		return ordinalMap[digit1]
	return "th"
```